torch/nn/modules/conv.py

```diff
@@ -109,6 +123,8 @@ class Conv1d(_ConvNd):
         dilation (int or tuple, optional): Spacing between kernel elements
         groups (int, optional): Number of blocked connections from input channels to output channels
         bias (bool, optional): If True, adds a learnable bias to the output
+        initializer(dict, optional): dictionary of initializer of weights and bias, if None (default),
```

Comment:
Thank you for advice, is `dictionary of initializer of weights and bias (for instance {"weight": lambda x: init.kaiming_uniform(x)}), if None (default), they are uniformly initialized.` enough?